.travis.yml

```diff
@@ -1,4 +1,25 @@
 language: ruby
+
+sudo: false
+
+cache: bundler
+
 rvm:
-  - 1.9.3
-before_install: gem install bundler -v 1.11.2
+  - 2.0.0
+  - 2.1
+  - 2.2
+  - 2.3.0
+  - ruby-head
```

Comment:
We test against ruby-head but alllow failures, it's a canary technique that allows you to allow for future proofing without breaking everything 👍